depends/packages/qt.mk

```diff
@@ -261,9 +262,6 @@ define $(package)_preprocess_cmds
 endef
 
 define $(package)_config_cmds
-  export PKG_CONFIG_SYSROOT_DIR=/ && \
```

Comment:
fwiw: moving this to depends/funcs.mk broke my Nix depends build. I'm not sure why.

https://github.com/0xB10C/bitcoind-gunix/issues/1#issuecomment-1714381895